.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -96,6 +96,19 @@ jobs:
           NUM_OF_PAGES=$(pdfinfo main_${{ matrix.language }}.pdf | awk '/^Pages:/ {print $2}')
           echo "NUM_OF_PAGES=${NUM_OF_PAGES}" >> "$GITHUB_OUTPUT"
 
+      - name: Check for broken CMap
+        if: github.event_name == 'pull_request'
+        run: |
+          QUOTE=$(grep -A1 heegusquote main_${{ matrix.language }}.tex | tail -1 | grep -o "{.*}" | tail -c +2 | head -c -2)
```

Comment:
this tail|head deserves a comment

Reply:
Either that, or maybe you know a better way of getting rid of the {} surrounding the text? Not a fan of this tail|head.